.gitmodules

```diff
@@ -102,3 +102,7 @@
 [submodule "third-party/dr_libs/src"]
 	path = third-party/dr_libs/src
 	url = https://github.com/mudita/dr_libs.git
+[submodule "reliance-edge"]
+	path = third-party/reedgefs/src
+	url = ../reliance-edge.git
```

Comment:
could you please use full URL for submodule?

Reply:
Most of mudita forkes got relative paths.

If we need to use full paths let's cover all in separate jira case